redhat-appstudio/remote-secret
# Add a flag to disable HTTP/2 in the webhooks and kube-rbac-proxy

### What does this PR do?
`$TITLE`

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-659

### How to test this PR?
1. Deploy the remote secrets operator
2. Take note of the IP address of the remote secret webhook server: `kubectl -n remotesecret exec deployment/remote-secret-controller-manager -t -- env | grep WEBHOOK_SERVICE_SERVICE_HOST | cut -d= -f2`
3. Run curl prompt in the cluster: `kubectl run curl --image=curlimages/curl -t -i -- sh`
4. in that prompt, run this command:
   ```
   curl -kv https://<...IP noted above...>:443
   ```
5. Make sure that the request is using HTTP/1.1

Test image: `quay.io/lkrejci/remote-secret-controller:svpi-659`
